packages/backend/src/controllers/schedulerController.ts

```diff
@@ -111,6 +111,36 @@ export class SchedulerController extends Controller {
         };
     }
 
+    /**
+     * Set scheduler enabled
+     * @param schedulerUuid The uuid of the scheduler to update
+     * @param req express request
+     * @param body the enabled flag
+     */
+    @Middlewares([
+        allowApiKeyAuthentication,
+        isAuthenticated,
+        unauthorisedInDemo,
+    ])
+    @SuccessResponse('201', 'Updated')
+    @Patch('{schedulerUuid}/enabled')
```

Comment:
You kind of alluded to this in one of your comments, but would it make sense to make the least the name of this endpoint more generic? `'{schedulerUuid}/control'` or something. So that we send other control signals (pause once maybe) to the same endpoint. Or should we just use the generic scheduler patch endpoint? 

Those suggestions have their downsides too -- if you did it this way for a specific reason, that's cool with me.

Reply:
>  Or should we just use the generic scheduler patch endpoint?

I already made a comment about this here https://github.com/lightdash/lightdash/pull/8996#discussion_r1494785458

`/control` could work, but I don't think we are going to add more control flags in the future (even `notify me once` I think it will be part of the `generic patch` endpoint ) 

So I called it /enabled so we can be explicit about what this does... 

But I have no strong opinion, so let me know what you think.

Reply:
Right, I read your comment, then forgot to look back at it. 

Personally I'd expect it to be part of the generic update endpoint. But I see your point about it being annoying to pass the whole object to change this. The tradeoff is having this extra endpoint to maintain. 

I think I'd lean towards using /update, but I don't think I have a strong enough feeling about it to ask you to change it 😄